packages/maker.js/test/svg.js

```diff
@@ -101,7 +101,7 @@ describe('Export SVG', function () {
         const svg = makerjs.exporter.toSVG(model, exportOptions);
         const expected = [
             '<svg width="50" height="50" viewBox="0 0 50 50" xmlns="http://www.w3.org/2000/svg">',
-                '<g id="svgGroup" stroke-linecap="round" fill-rule="evenodd" font-size="8pt" stroke="#333" stroke-width="0.5mm" fill="green" style="stroke:#333;stroke-width:0.5mm;fill:green">',
+                '<g id="svgGroup" stroke-linecap="round" fill-rule="evenodd" font-size="8pt" stroke="#333" stroke-width="0.5mm" fill="green" style="stroke:#333;stroke-width:0.5mm;fill:green" class="">',
```

Comment:
Let's add `className="test"` to `exportOptions` and then put `class="test"` here.

Reply:
Added and tested with `className="test"`